modules/microshift-updating-rpms-y.adoc

```diff
@@ -22,14 +27,35 @@ You cannot downgrade {microshift-short} with this process. Downgrades are not su
 
 .Procedure
 
-. Enable the {microshift-short} repositories by running the following command:
+. For all lifecycles, enable the repository for your release by running the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ sudo subscription-manager repos \
-    --enable rhocp-{ocp-version}-for-{rhel-major}-$(uname -m)-rpms \
-    --enable fast-datapath-for-{rhel-major}-$(uname -m)-rpms
+    --enable rhocp-<4.18>-for-<9>-$(uname -m)-rpms \ # <1>
+    --enable fast-datapath-for-<9>-$(uname -m)-rpms # <2>
+----
+<1> Replace _<4.18>_ and _<9>_ with the compatible versions of your {microshift-short} and {op-system-base-full}.
+<2> Replace  _<9>_ with the compatible version of {op-system-base}.
+
+. For extended support (EUS) releases, also enable the EUS repositories by running the following command:
++
+[source,terminal]
+----
+`$ sudo subscription-manager repos \
+    --enable rhel-<9>-for-x86_64-appstream-eus-rpms \ # <1>
+    --enable rhel-<9>-for-x86_64-baseos-eus-rpms` # <2>
+----
+<1> Replace _<9>_ with the compatible major version number of {op-system-base}.
+<2> Replace _<9>_ with the compatible major version number of {op-system-base}.
+
+. Avoid unintended future updates into an unsupported configuration by locking your operating system version with the following command:
```

Comment:
Change here if you made the change up [there](https://github.com/openshift/openshift-docs/pull/87660/files#r1940126276).